src/Elasticsearch/Connections/Connection.php

```diff
@@ -168,6 +168,11 @@ public function __construct(
         }
         $port = $hostDetails['port'];
 
+        if (isset($connectionParams['client']['port_in_header']) && $connectionParams['client']['port_in_header']) {
+            if (!in_array((int) $port, [80,443])) {
```

Comment:
I think it should be possible to have port 80 or 443 in the header as well. It is perfectly acceptable according to spec, so I don't see a need for special treatment.

Reply:
Including the port is ok according to spec, and those who do not want the port in the Host header can just set the newly introduced option to false to disable it.

Reply:
I removed the default ports because it's a common standard in many HTTP client. Moreover, I suspect this will fix also #782.

Reply:
I don't feel super strongly about this, but my reasoning goes like this:
1. The elasticsearch client adds the port to the Host header, as is allowed by the HTTP spec.
2.  Somebody will encounter #548, for which the easy solution is to not append the port to the Host header (they could also fix the broken proxy server, but lets be realistic).
3.  They can accomplish this with `includePortInHostHeader(false)`.

So this PR solves #782 even if you don't exclude port 80 and 443.

Anyway, you decide :)

Reply:
:thinking: Ok, you convinced me. I'll remove this standard port check from the PR. I'll investigate more on that for the next major version of elasticsearch-php. Thanks again for your precious feedback.